lib/apollo-federation/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ApolloFederation
-  VERSION = '3.8.5'
```

Comment:
Do you think it makes any sense? 
I mean we didn't publish it as a gem and this can add confusion/conflicts later when apollo update gem on they side.

Reply:
not sure, but at the same time just not to mix up this version with the original one
their versioning approach isn't clear so hard to say what is the correct way to maintain the version|

tl;tr
it's kind of stub now